src/sentry/receivers/onboarding.py

```diff
@@ -422,3 +423,29 @@ def record_issue_tracker_used(plugin, project, user, **kwargs):
         project_id=project.id,
         issue_tracker=plugin.slug,
     )
+
+
+@integration_added.connect(weak=False)
+def record_integration_added(integration, organization, user, **kwargs):
+    task = OrganizationOnboardingTask.objects.filter(
+        organization_id=organization.id,
+        task=OnboardingTask.INTEGRATIONS,
+    ).first()
```

Comment:
@Neo-Zhixing This is a great case where you can use `get_or_create` to reduce some of the duplication

Reply:
Nope, can't do that.
When we create, we always create the same item.
When we update, the updated item relies on the previous state.
These are two distinct behaviors that are not covered by get or create.

Reply:
@Neo-Zhixing can you use an F expression? https://docs.djangoproject.com/en/4.0/ref/models/expressions/#f-expressions

Reply:
No - we would've been able to do that if this is a jsonb column. But it's saved as text in the database so PostgreSQL wouldn't know what to do with it